src/screens/EncounterScannerScreen.tsx

```diff
@@ -0,0 +1,137 @@
+import { BarCodeScanningResult, Camera } from 'expo-camera';
+import * as Linking from 'expo-linking';
+import React, { useEffect, useState } from 'react';
+import { useCallback } from 'react';
+import { Alert, ScrollView, StyleSheet } from 'react-native';
+
+import appJson from '../../app.json';
+import {
+  BoldText,
+  Button,
+  RegularText,
+  SectionHeader,
+  Wrapper,
+  WrapperWithOrientation
+} from '../components';
+import { LoadingSpinner } from '../components/LoadingSpinner';
+import { device, texts } from '../config';
+import { useCreateEncounter, useEncounterUser } from '../hooks';
+import { User } from '../types';
+
+const showErrorAlert = () =>
+  Alert.alert(texts.encounter.errorScanTitle, texts.encounter.errorScanBody);
+
+const parseQrCode = (data: string): string | undefined => {
+  const result = Linking.parse(data);
+
+  if (result.scheme === (__DEV__ ? 'exp' : appJson.expo.scheme) && result.path === 'encounter') {
```

Comment:
i miss an explanation for `exp`. why this in dev mode? i have a clue but cannot be sure.

Reply:
i added a comment for explanation